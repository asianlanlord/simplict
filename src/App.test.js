import renderer from 'react-test-renderer';

import Timer from './Components/timer.js';
import Stats from './Components/statistics.js';
import Scrambler from './Components/scrambler.js';

it('Timer renders correctly', () => {
    const tree = renderer.create(<Timer />).toJSON();
    expect(tree).toMatchSnapshot();
});


it('Stats renders correctly', () => {
    const tree = renderer.create(<Stats />).toJSON();
    expect(tree).toMatchSnapshot();
});


it('Scrambler renders correctly', () => {
    const tree = renderer.create(<Scrambler />).toJSON();
    expect(tree).toMatchSnapshot();
});
