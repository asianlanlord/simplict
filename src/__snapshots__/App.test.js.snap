// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scrambler renders correctly 1`] = `
<div
  id="scramble"
>
  L' D' F' R2 B U2 D2 R' B2 U F' L D' U R' B F' B2 R2 U2 F2 L' D'
  <img
    alt="refresh_button"
    id="refresh_icon"
    onClick={[Function]}
    src="Refresh_icon.png"
  />
</div>
`;
